pyproject.toml

```diff
@@ -47,6 +47,7 @@ pytest = "^6.0"
 pylint = "*"
 guzzle_sphinx_theme = "*"
 recommonmark = "*"
+matplotlib = "^3.3.2"
```

Comment:
Do we really want to add a new dependency?
I like the matplotlib library, however, manim already uses PIL for dealing with images, so why not use 
https://pillow.readthedocs.io/en/stable/reference/Image.html#functions  `PIL.Image.open` for reading the image?

Reply:
Ah, sorry, I did not read the whole thread, thanks for pointing out that matplotlib needs to be added eighter way